distribution/receiver/routes/send-notification-email-route.xml

```diff
@@ -0,0 +1,20 @@
+<routes xmlns="http://camel.apache.org/schema/spring">
+    <route id="send-notification-email-route">
+       <!-- <from uri="scheduler:send-notification-email-route?initialDelay=1000&amp;delay=60000" /> -->
```

Comment:
You can remove this line instead of commenting it out

Reply:
Ok